extensions/mssql/src/contracts.ts

```diff
@@ -1684,10 +1684,18 @@ export interface GetDataFolderRequestParams {
 	connectionUri: string;
 }
 
+export interface GetBackupFolderRequestParams {
```

Comment:
Small nit, but could you move this interface to just above GetBackupFolderRequest to keep the calls organized?

Reply:
Moved!